presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -208,6 +210,34 @@ public void testLongStrideDictionary()
         testRoundTripNumeric(concat(ImmutableList.of(1), nCopies(9999, 123), ImmutableList.of(2), nCopies(9999, 123)), BigintRange.of(123, 123, true));
     }
 
+    @Test
+    public void testFloats()
+            throws Exception
+    {
+        TupleDomainFilter floatFilter = FloatRange.of(Float.MIN_VALUE, false, true, Float.MAX_VALUE, false, true, true);
```

Comment:
> "It is confusing to me to have a filter that matches everything."
changed it. I copied these tests from the TestOrcReader. I should have modified them instead of trying to keep them same.

Reply:
added a test to cover the position > streamPosition